tests/headers/CMakeLists.txt

```diff
@@ -13,13 +13,7 @@ set(exclude_from_all_headers)
 # These files are either known to expose incomplete types or are known to
 # require special handling to avoid circular #include dependencies.
 set(exclude_from_headers
-  "${PROJECT_SOURCE_DIR}/hpx/include/compression.hpp"
```

Comment:
So this PR will stop testing the headers from the former `hpx/include` directory. Is that intentional?

Reply:
They're tested as part of the module (hence no need to exclude these two here; they're excluded in the module instead). See https://github.com/STEllAR-GROUP/hpx/blob/264de211ae34a3a784324b52599c5f0b525b4dba/libs/include/tests/CMakeLists.txt#L34-L43.

Reply:
Ahh, I did miss that! Thanks!